examples/react-app/package.json

```diff
@@ -12,6 +12,7 @@
     "@iot-app-kit/source-iotsitewise": "*",
     "@iot-app-kit/source-iottwinmaker": "*",
     "@iot-app-kit/table": "*",
+    "cytoscape": "^3.25.0",
```

Comment:
Why do we need to install cytoscape in the react-app ?

Reply:
I am sorry but this is not acceptable at all and we agreed before we don't want the customer to know or rely on our choice of graph viz library. This will really tie us to this library.